Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -7,6 +7,9 @@
 import Foundation
 
 internal class RUMViewScope: RUMScope, RUMContextProvider {
+    // MARK: - Background Scope
+    static let RUM_BACKGROUND_VIEW_URL = "com/datadog/background/view"
+    static let RUM_BACKGROUND_VIEW_NAME = "Background"
```

Comment:
✏️ We don't use capitals for defining constants in Swift. Also, in `dd-sdk-ios` we adopt the convention of grouping constants in `Constants` struct acting as a namespace:
```swift
internal class RUMViewScope: RUMScope, RUMContextProvider {
   struct Constants {
      static let backgroundViewURL = "com/datadog/background/view"
      static let backgroundViewName = "Background"
   }
   
   // ...
}
```